### ISSUE TITLE
ImportError: cannot import name simplejson

### ISSUE THREAD

**Issue by jstncno** [RyanBalfanz/django-sendgrid#72]
Can't import simplejson from from django.utils for Django 1.7...
I noticed the development branch fixes this. Can that be uploaded to PyPI so I can easily pip install it?

**Comment by cheenbabes** [comment]
Yeah, I second this. Running `pip install django-sendgrid` downloads the code with simplejson errors

**Comment by cheenbabes** [comment]
@CrashPhoenix  for now you can just copy this https://github.com/RyanBalfanz/django-sendgrid/blob/develop/sendgrid/mixins.py to replace the mixins.py that ships from PyPi. That fixed it for me and my project built.

**Comment by ArchTaqi** [comment]
I am also facing the same issue in `django 1.8`

**Comment by nyxcharon** [comment]
+1. This also effects me.

**Comment by mezitax** [comment]
+1 any solution?

**Comment by cheenbabes** [comment]
It's an import error - you can fix it by copying the file I mentioned above.
But someone should really submit a PR and get this fixed. Or use Mandrill -- I've found it much more developer friendly.

**Comment by lucasnoah** [comment]
I get this error as well

**Comment by ArchTaqi** [comment]
It's still not solved.

**Comment by emojiface** [comment]
Getting this error

**Comment by jmnyarega** [comment]
i get the error, any fix?

**Comment by s-surineni** [comment]
still now solved
